monitors.yml

```diff
@@ -1,24 +1,142 @@
 defaults:
   monitors:
-    multi: true
     options:
       locked: true
+      new_host_delay: 300
+      no_data_timeframe: 2
+      notify_audit: false
+      renotify_interval: 0
+      silenced: {}
+      timeout_h: 0
```

Comment:
Do all of these make sense to be in the global options like this? 

I was thinking that maybe we could do additional options by tag or something later, but for now being unset should be the default for most of these right?